datalake/backend/parameter_manager.py

```diff
@@ -0,0 +1,21 @@
+import os
+from enum import Enum
+
+import boto3
+
+ssm_client = boto3.client("ssm")
+
+ENV = os.environ["DEPLOY_ENV"]
+
+
+class ResourceName(Enum):
+    DatasetsTableName = "DatasetsTableName"
```

Comment:
If we use `DatasetsTableName = auto()` we can use for example `ResourceName.DatasetsTableName.name` to get the string name of that entry.

Also, the entry names should probably be `UPPER_CASE` since they are effectively constants.